cmd/mtaBuild.go

```diff
@@ -196,8 +196,11 @@ func runMtaBuild(config mtaBuildOptions,
 		if err != nil {
 			return err
 		}
+
+		var packageJSONFiles []string
+		packageJSONFiles, err = npmExecutor.FindPackageJSONFiles(nil)
```

Comment:
Oh I didn't even intend to ignore the error here, but I also see your point. So you suggest to have a `FindPackageJSONFilesWithExcludes()` which contains the current implementation and another function `FindPackageJSONFiles()` which simply calls `FindPackageJSONFilesWithExcludes(nil)` and ignores the error?

Reply:
Exactly that, yes.

Reply:
It's just a suggestion, of course.

Reply:
Was this addressed now? I would also prefer having an alternative method as you do not have to pass nil. Alternatively you could also pass some an empty exclude list instead of nil.

Reply:
The latest commit addresses this.